redmi.md

```diff
@@ -1 +1 @@
-'master branch'
+'dvelop branch'
```

Comment:
We don't need this in readme file, It should details about project, in our case it can be problem statement.